experimental/dds/tree2/src/test/shared-tree/sharedTree.spec.ts

```diff
@@ -647,258 +667,271 @@ describe("SharedTree", () => {
 		});
 
 		it("does not undo edits made remotely", () => {
-			const value = "42";
 			const provider = new TestTreeProviderLite(2);
-			const [tree1, tree2] = provider.trees;
-
-			const initialState: JsonableTree = {
-				type: brand("TestValue"),
-				value: "A",
+			const content: InitializeAndSchematizeConfiguration = {
+				schema: jsonSequenceRootSchema,
+				allowedSchemaModifications: AllowedUpdateType.None,
+				initialTree: ["tree2"],
 			};
-			initializeTestTree(tree2, initialState);
+			// Do initialization on tree2
+			const tree2 = provider.trees[1].schematize(content);
+			provider.processMessages();
+			const tree1 = provider.trees[0].schematize(content);
 			provider.processMessages();
 
-			assert.equal(getTestValue(tree1), "A");
+			validateRootField(tree1, ["tree2"]);
 
 			// Insert node
-			insert(tree1, 0, value);
+			insert(tree1, 0, "tree1");
 			provider.processMessages();
 
-			const testValuesAfterInsertion = getTestValues(tree1);
-			assert.equal(testValuesAfterInsertion[0], "A");
-			assert.equal(testValuesAfterInsertion[1], value);
+			// TODO: older versions of this test expected the revers order here, but its unclear why.
```

Comment:
I just looked into this. `getTestValues` reverses the order of the values for some reason.